package.json

```diff
@@ -5,18 +5,39 @@
   "description": "Looker extension examples",
   "author": "Looker",
   "private": true,
+  "workspaces": [
+    "react/javascript/*",
+    "react/typescript/*",
+    "vanilla/*"
+  ],
   "scripts": {
     "bootstrap": "lerna bootstrap",
     "lerna": "lerna",
     "clean": "lerna run --parallel clean",
     "build": "lerna run --parallel build",
     "lint": "lerna run --parallel lint:es",
+    "lint:fix": "lerna run --parallel lint:es:fix",
     "tsc": "lerna run --parallel tsc",
     "test": "lerna run --parallel test",
     "uberext": "ts-node ./scripts/create_uber_extension_project.ts",
+    "update_packages": "ts-node ./scripts/update_packages.ts",
+    "dev:demo-core-sdk": "cd react/javascript/demo-core-sdk && yarn develop",
     "dev:demo-embeds": "cd react/javascript/demo-embeds && yarn develop",
+    "dev:demo-extension-sdk": "cd react/javascript/demo-extension-sdk && yarn develop",
```

Comment:
Could this use the

```sh
yarn workspace <workspace> develop
```

pattern instead, or are these still not workspaces?

Reply:
Was not aware of this. Will probably try out and put up as a separate PR

Reply:
Okay, fixed it